dev/docker/README.md

```diff
@@ -0,0 +1,86 @@
+Apache REEF (incubator) Docker Tests
+====================================
+
+[Apache REEF (incubator)](http://reef.incubator.apache.org/) is 
+an approach to simplify and unify the lower layers of big data systems
+on modern resource managers. This project maintains docker images
+to simulate the target underlying systems as similar as possible.
+
+Docker-based Test Cluster
+-------------------------
+
+| OS           | Java            | Hadoop Version            | Description   | alias     |
```

Comment:
Couldn't we somehow reduce the size of the fields?
For example we can remove `Oracle` and make the first field `Environment`
What's `Description=HDInsight 3.1`? We're not testing HDInsight, the runtime are we?(I'm actually not familiar with the relationship between YARN and HDInsight) Then maybe we can remove this field?